k8s-openapi-codegen-common/src/templates/impl_deserialize.rs

```diff
@@ -69,7 +69,7 @@ pub(crate) fn generate(
             flattened_field = Some((field_name, field_type_name));
 
             writeln!(field_value_defs,
-                r#"                let mut value_{field_name}: std::collections::BTreeMap<{local}serde_value::Value, {local}serde_value::Value> = Default::default();"#)?;
+                r#"                let mut value_{field_name}: {local}serde_json::Map<String, {local}serde_json::Value> = Default::default();"#)?;
```

Comment:
Keep it as `std::collections::BTreeMap` for now. I assume you will send a later PR to replace all `std::collections::BTreeMap`, so this can be changed in that.

Reply:
I have a question. `std::collections::BTreeMap<_, _>` does not implement `serde::Deserializer<'_>`. Then is it good to wrap it with the other types (e.g. `serde_json::Map<_, _>`?

Reply:
Replacing `std::collections::BTreeMap` should be available after merging https://github.com/serde-rs/json/pull/1228.

Reply:
> I have a question. `std::collections::BTreeMap<_, _>` does not implement `serde::Deserializer<'_>`. Then is it good to wrap it with the other types (e.g. `serde_json::Map<_, _>`?
> 
> Or there is another way: replace `std::collections::BTreeMap` into `serde_json::Map` now (in this PR).

Replacing is ready (code ready)!

Reply:
> I have a question. `std::collections::BTreeMap<_, _>` does not implement `serde::Deserializer<'_>`. Then is it good to wrap it with the other types (e.g. `serde_json::Map<_, _>`?
> 
> Or there is another way: replace `std::collections::BTreeMap` into `serde_json::Map` now (in this PR).

It's denied, so replacing without any stuff would be impossible.

Reply:
> Replacing `std::collections::BTreeMap` should be available after merging [serde-rs/json#1228](https://github.com/serde-rs/json/pull/1228).

It's denied, so replacing without any stuff would be impossible.

Reply:
Oh right, I missed that this is used as `Deserializer` later. Then using `serde_json::Map` here is fine.

Reply:
I see!